book/modules/module14-exercises.tex

```diff
@@ -82,6 +82,30 @@
 
 			\item What is $\Null(A^{-1})$? Justify your answer.
 		\end{enumerate}
+\prob Anna and Ella are studying relations between the determinant and the volume. In particular, they are studying $\mathcal{S}:\mathbb{R}^3\rightarrow\mathbb{R}^3$, where $\mathcal{S}\matc{x \\ y \\ z}=\matc{4x \\ 2z \\ 0}$, and $\mathcal{T}:\mathbb{R}^3\rightarrow\mathbb{R}^2$, where $\mathcal{T}\matc{x \\ y \\ z}=\matc{2x \\ 8z}$.
+\begin{enumerate}
+	\item Anna says: \emph{since the image of $C_{3}$ under $\mathcal{S}$ is the
+		parallelepiped generated by
+		$\matc{4 \\ 0 \\ 0},\matc{0 \\ 0 \\ 0}, and \matc{0 \\ 2 \\ 0}$, which is
+		2-dimensional parallelogram, the volume of $\mathcal{S}(C_{3})$ is just
+		the area of this parallelogram, which is 8. Thus, $\det(\mathcal{S})=8$.}
+		Ella disagrees with Anna. She claims: \emph{$\det(\mathcal{S})$ is
+		undefined, because $\mathcal{S}$ is not invertible.} Do you think any of
+		their arguments are correct? If not, point out the mistake each of them made
+		and determine $\det(\mathcal{S})$ (give a number or explain why it is
+		undefined).
```

Comment:
This is great!

Switch the `\emph{...}` to be around the names (e.g., `\emph{Anna says}:`) instead of around the description. Put each person's description in a new paragraph.